core/content/starter.js

```diff
@@ -34,7 +34,11 @@
 	if (Connector.playerSelector !== null) {
 		console.log('Web Scrobbler: Setting up observer');
 
-		var observer = new MutationObserver(Connector.onStateChanged);
+		var observer = new window.MutationObserver(function () {
```

Comment:
jshint checks for undefined variables.  It wouldn't pass the grunt test without including window.